src/mod_roster_rdbms.erl

```diff
@@ -145,22 +173,13 @@ get_roster_entry(LUser, LServer, LJID, full) ->
     end.
 
 get_roster_entry_t(LUser, LServer, LJID, full) ->
-    case get_roster_entry_t(LUser, LServer, LJID) of
-        does_not_exist -> does_not_exist;
-        Rentry ->
-        case read_subscription_and_groups_t(LUser, LServer, LJID) of
-            error -> error;
-            {Subscription, Groups} ->
-                Rentry#roster{subscription = Subscription, groups = Groups}
-        end
-    end.
-
+    get_roster_entry(LUser, LServer, LJID, full).
```

Comment:
My question about the transaction context remains valid here. I am starting to think it deserves an explanation in the commit message - the motivation and reasoning behind removing these `_t` versions.